internal/transfers/transfers.go

```diff
@@ -185,14 +188,48 @@ func getTransferID(r *http.Request) id.Transfer {
 	return id.Transfer("")
 }
 
+type transferFilterParams struct {
+	StartDate time.Time
+	EndDate   time.Time
+	Limit     int64
+	Offset    int64
+}
+
+func readTransferFilterParams(r *http.Request) transferFilterParams {
+	params := transferFilterParams{
+		StartDate: time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC),
+		EndDate:   time.Now().Add(24 * time.Hour),
+		Limit:     100,
+		Offset:    0,
+	}
+	if r == nil {
+		return params
+	}
+	if v := r.URL.Query().Get("startDate"); v != "" {
+		params.StartDate = util.FirstParsedTime(v, base.ISO8601Format, util.YYMMDDTimeFormat)
+	}
+	if v := r.URL.Query().Get("endDate"); v != "" {
+		params.EndDate, _ = time.Parse(base.ISO8601Format, v)
+		fmt.Printf("params.EndDate=%v\n", params.EndDate)
+	}
+	if v := r.URL.Query().Get("limit"); v != "" {
+		params.Limit, _ = strconv.ParseInt(v, 10, 32)
```

Comment:
I'm going to wrap these with a hard limit so we don't page tons and tons of objects back.